extensions/README.md

```diff
@@ -0,0 +1,37 @@
+# NEAR CLI extensions system
+`NEAR CLI` is built to scale. The number of possible features is endless. Instead of choosing only some of them, we are creating an `Extensions System` that will empower our users to choose, build and share `NEAR CLI` functionality.
+
+## How it works
+Extensibility is achieved by translating a `NEAR CLI` invocation of the form `near (?<command>[^ ]+)` into an invocation of an external tool `near-${command}` that then needs to be present in one of the user's `$PATH` directories.
+It means that you can write it in any language and with the use of any framework, it just needs to be called `near-cli-{extension-name}` and be installed on your system. This approach is inspired by [Cargo](https://github.com/rust-lang/cargo).
+
+## How to build an extension
+As mentioned above, any binary can become an extension, but we are encouraging developers to use [Rust](https://www.rust-lang.org/), [Clap](https://docs.rs/clap/2.33.0/clap/), and a set of libraries developed by NEAR. Here is some of them:
+- `near-cli-builder` - CLI specific helpers to make your life easier and follow the standards of `NEAR CLI` at the same time (NOTE: Under development)
+- `near-api-rs` - Rust library to interact with accounts and smart contracts on NEAR. (NOTE: Under development)
+- [near-jsonrpc-client-rs](https://github.com/near/near-jsonrpc-client-rs) - Lower-level JSON RPC API for interfacing with the NEAR Protocol.
+
+## Example
+Core `NEAR CLI` does not have validator specific functionality, but we can add it as a simple bash script:
```

Comment:
I believe it is worth mentioning the name of the script (`near-staking-pool-info` [without any extension on *nix systems]) and the permissions to execute.